debezium-core/src/test/java/io/debezium/transforms/UnwrapFromEnvelopeTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.Test;
+
+import io.debezium.data.Envelope;
+
+/**
+ * @author Jiri Pechanec
+ */
+public class UnwrapFromEnvelopeTest {
+
+    private static final String DROP_DELETES = "drop.deletes";
+    private static final String DROP_TOMBSTONES = "drop.tombstones";
```

Comment:
In this case this is intentional. If somebody by accident changes the prop names then the test would detect it, otherwise it could happily end up in releases.